packages/three-vrm-materials-mtoon/src/utils/getTextureColorSpace.ts

```diff
@@ -1,7 +1,9 @@
 import * as THREE from 'three';
 
 const encodingColorSpaceMap: Record<any, '' | 'srgb'> = {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
it's possible to do this in eslint config but needs some regex magic. I think it's easier to disable here.